packages/frontend/src/components/create-campaign/form/index.tsx

```diff
@@ -51,8 +52,7 @@ function validatePayload(
     } else return null;
 
     return new CampaignPreviewPayload(
-        payload.dex,
-        payload.pool,
+        payload.target,
```

Comment:
Shouldn't we validate the target here? Exactly like it is done for the distributables. Which btw could become a new entry in the campaign payload to mimic what is done in the SDK for mapping purposes. So instead of having `points`, `fees` and `tokens` or whatever, we would have a single `distributables` key with a `type` discriminant and a type-specific payload.

Reply:
We are checking for the existence of the `target` above, but yeah for the `distributables` it would make sense.

Reply:
Yeah existence should be checked, but the single target should be validated depending on its type, internally, too.